src/crafting_gui.cpp

```diff
@@ -591,100 +624,14 @@ static void draw_recipe_subtabs( WINDOW *w, std::string tab, std::string subtab,
         mvwputch( w, i, width - 1, BORDER_COLOR,  LINE_XOXO ); // ^|
     }
 
-    switch( mode ) {
```

Comment:
The space after ```switch``` is 1 that I didn't keep correctly during the merge. Since it was only 1 I said, f**k it, I'll change it later :P